backend/api/src/create-market.ts

```diff
@@ -128,8 +131,18 @@ export async function createMarketHelper(body: schema, auth: AuthedUser) {
     ante || 0
   )
 
-  if (group && groupId)
+  if (group && groupId) {
     await addGroupContract(groupId, group, contractRef, userId)
+    if (contract.visibility == 'private') {
+      const contractCreator = await getUser(contract.creatorId)
+      if (!contractCreator) throw new Error('Could not find contract creator')
+      await createNewContractInFromPrivateGroupNotification(
+        contractCreator,
+        contract,
+        group
+      )
```

Comment:
Sending emails and such is generally better to do in a trigger (i.e. onCreateContract), so it does not slow down the initial request to create the contract.

I would probably move this logic there (to `on-create-contract.ts`)

Reply:
That's where I originally had it (this is what took me a long time to debug yesterday), but groups aren't populated when on-create-contract is triggered yet